recipe-model.js

```diff
@@ -0,0 +1,39 @@
+const mongoose = require("mongoose");
+
+//Schema constructor from Mongoose
+const Schema = mongoose.Schema;
+
+//use the schema constructor to create our cat schema
+const recipeSchema = new Schema({
+  title: {
+    type: String, 
+    required: true,
+    unique: true,
+  },
+  level: {type: String},
+  ingredients: [ String ], 
```

Comment:
Here you are saying that ingredients must be an array of Strings, you are being more specific that what was asked in the lab, this is okay but keep in mind that Array is also a javascript type and you can write : 
```
 ingredients: Array,
```